myapp/app/helpers/sessions_helper.rb

```diff
@@ -11,6 +11,10 @@ def logged_in?
     current_user.present?
   end
 
+  def admin_logged_in?
+    logged_in? && current_user&.is_admin?
```

Comment:
Since  `logged_in?` is true `current_user` cannot be nil. So the `&` is not necessary.
```suggestion
    logged_in? && current_user.is_admin?
```

Reply:
@zackey-y 
Thank you. I fixed that. https://github.com/Fablic/training/commit/39e025da68afc4b5b9cb439fc7536e9c35c2345e